### ISSUE TITLE
Separate settings.json and board status [proposal]

### ISSUE THREAD

**Issue by KostyaEsmukov** [MDSLab/s4t-lightning-rod#26]
Hello,

I propose to move board status props out of settings.json to a separate file. This would make configuration cleaner (w/o confusing props such as config:board:position , config:board:status , etc) and would eliminate need to ever write to the configuration file.
Please let me know what do you think about it.

Kostya.

**Comment by flongo82** [comment]
This should be okay, even if @fabio14 is the best person to answer here because he is implementing the version of the Lightning-rod which is compatible with OpenStack and he implemented some functions to change the configuration file from the Cloud. @fabio14 let us know what you think about this.

**Comment by fabio14** [comment]
it is ok for me, maybe a _properties.json_? 
But we have to distinguish which ones are proprs and which ones are settings definitively

**Comment by KostyaEsmukov** [comment]
*Props*, IMO, are the things that should not be ever touched by the user.

Are there such settings that should be changeable from the cloud, that are not *props*?

**Comment by flongo82** [comment]
Problem is that the Cloud should be able to touch everything. I would say that maybe is better to separate configuration properties on the base of the context. For example networking configuration separated from drivers configuration and so on. But this is already there if I'm not wrong. @npeditto ?

**Comment by flongo82** [comment]
Talking with Nicola and Fabio we recalled that when starting implementing the OpenStack version of the architecture we decided to put everything in one single configuration file. 

In such a way, as soon as a board come up again after a disconnection or a failure can perform an hash of its configuration file, send it to the cloud in order to understand if it needs any update in the configuration. The cloud can perform the hash on its side and if it is different it can send the new configuration to the board without needing to understand what has been changed.
